extension/src/test/suite/util.ts

```diff
@@ -330,7 +330,7 @@ export const waitForSpyCall = async (
   messageSpy: SinonSpy,
   originalCallCount: number
 ): Promise<unknown> => {
-  await Time.delay(500)
+  await Time.delay(100)
```

Comment:
[F] Could probably go as far as using process.nextTick here but this is good enough for now.